lib/galaxy/tools/deps/conda_util.py

```diff
@@ -84,18 +98,31 @@ def __init__(self, conda_prefix=None, conda_exec=None,
         self.ensured_channels = False
         self._conda_version = None
         self._miniconda_version = None
+        self._conda_build_available = None
+        self._use_local = use_local
```

Comment:
What about just use:
```python
        self.use_local = use_local
```
and remove the property below? Not sure it's worth the complication since it's never changed after the initialisation.